### DISCUSSION TITLE
PG matrix with non normalized data

### DISCUSSION THREAD

**Discussion by jmsanfrutos** [vdemichev/DiaNN#426] [Q&A]
Hi there,

Thank you so much for the great software. 

I was wondering if there is a way to get the pg matrix without MaxLFQ data, but with non normalized data instead. 

I have used the command --no-maxlfq but the pg.matrix is empity and I would like to know if there is any comand that allow to include the PG.quantity in this table.

Thank you very much in advanced.

BW

Julia

**Comment by vdemichev** [comment]
Hi Julia,

MaxLFQ is protein quant, not normalisation. You can still use MaxLFQ without normalisation. For this, please just disable cross-run normalisation in DIA-NN GUI.

Best,
Vadim

**Reply by jmsanfrutos** [reply]
Dear Vadim,

Thank you so much for your quick reply. I just realized that I was not clear in my question, I would like to know if there is any command that allows to get in the pg.matrix the values from the PG.Quantity instead fo MaxLFQ.

Thanks!

Julia

**Reply by vdemichev** [reply]
Hi Julia, can do this by disabling MaxLFQ with --no-maxlfq, but I don't recommend this.

Best,
Vadim
